packages/inventory-compliance/src/ComplianceRemediationButton/ComplianceRemediationButton.js

```diff
@@ -97,6 +103,7 @@ ComplianceRemediationButton.propTypes = {
         profiles: propTypes.arrayOf(propTypes.shape({
             refId: propTypes.string,
             name: propTypes.string,
+            supported: propTypes.bool.isRequired,
```

Comment:
Shouldn't this be up one more level under `allSystems`?

Reply:
Yep, you were right. Moved it up. It should now be correct.